send_email.py

```diff
@@ -0,0 +1,37 @@
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+
+sender_from_email = "sender@gmail.com"
+receiver_to_email = "receiver@gmail.com"
+password = input("Type your password and hit enter:")
+
+message = MIMEMultipart("alternative")
+message["Subject"] = "Network Analysis Report"
+message["From"] = sender_from_email
```

Comment:
It would be great if the user would be promted to enter his (google) email information as well as the reciever of the report